src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        setDescription(description);
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setDone(boolean done) {
```

Comment:
```suggestion 
 public void setDone(boolean isDone) {
```

Perhaps you can change the naming of the boolean input to the above?